content/docs/ids.md

```diff
@@ -45,15 +45,14 @@ The next three commands use the ID numbers shown in the report.
 The last command shows ID numbers again.
 Clearly, while the middle three commands are being run, it would be a mistake to renumber any tasks.
 When the last command is run, new ID numbers are shown, and any subsequent commands would use those.
-This is why only commands that display ID numbers perform a gc first.
+This is why only reports that display ID numbers perform a rebuild first.
 
 ## Configuration
 
-The `gc` configuration setting may be set to 'off' to disable the gc operation.
-This has the effect of letting data accumulate in the `pending.data`
-file, and effectively making the ID numbers static.
+The `gc` configuration setting may be set to 'off' to disable the rebuild operation.
```

Comment:
I'm curious what your thoughts are here? The config uses `gc`, but we're trying to use "rebuild" everywhere else..

Reply:
I would prefer to rename the config from `gc` to something like `rebuildworkingset`. But this depends on how you want to handle backwards compatibility.

Reply:
Let's leave this as-is, and please open a bug in GothenburgBitFactory/taskwarrior to rename the config option. We can keep the old one for backward compatibility.

Reply:
Sure! https://github.com/GothenburgBitFactory/taskwarrior/issues/3557